components/src/main/java/com/mercadopago/sdk/android/components/Label.kt

```diff
@@ -0,0 +1,437 @@
+package com.mercadopago.sdk.android.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mercadopago.sdk.android.foundation.theme.MercadoPagoTheme
+
+private const val TEXT_GROUP = "Text"
+
+/**
+ * Label Type enum class, used to determine the label type
+ * This its used to change the token color
+ */
+enum class LabelColorType {
+    /**
+     *  Primary: Label of the Primary type
+     */
+    Primary,
+
+    /**
+     *  Secondary: Label of the Secondary type
+     */
+    Secondary,
+
+    /**
+     *  Accent: Label of the Accent type
+     */
+    Accent,
+
+    /**
+     *  Negative: Label of the Negative type
+     */
+    Negative,
+
+    /**
+     *  Inverted: Label of the Inverted type
+     */
+    Inverted
+}
+
+/**
+ * Label Text type enum class, used to determine the label type
+ * This its used to change the text style
+ */
+enum class LabelTextType {
+    /**
+     * Tittle: Label of the Tittle type
+     */
+    Tittle,
```

Comment:
there is a typo here and on all the other title references since this should be Title
```suggestion
    Title,
```